docs/source/index.rst

```diff
@@ -140,15 +139,13 @@ More on how to get started with CleanVision:
    cleanvision/utils/index
 
 .. toctree::
-   :caption: Guides
    :hidden:
+   :maxdepth: 3
+   :caption: Tutorials
 
-   Tutorial Notebook <https://colab.research.google.com/github/cleanlab/cleanvision/blob/main/examples/tutorial.ipynb>
-   HuggingFace Notebook <https://colab.research.google.com/github/cleanlab/cleanvision/blob/main/examples/huggingface_dataset.ipynb>
-   Torchvision Notebook <https://colab.research.google.com/github/cleanlab/cleanvision/blob/main/examples/torchvision_dataset.ipynb>
-   Example Python Script <https://github.com/cleanlab/cleanvision/blob/main/examples/run.py>
-   More Example Notebooks <https://github.com/cleanlab/cleanvision-examples>
-   How To Contribute <https://github.com/cleanlab/cleanvision/blob/main/CONTRIBUTING.md>
+   tutorials/tutorial.ipynb
```

Comment:
in future consider renaming this to something like: `cleanvision_quickstart.ipynb`